tests/ci/run_posix_sanitizers.sh

```diff
@@ -7,8 +7,19 @@ source tests/ci/common_posix_setup.sh
 build_type=Release
 cflags=("-DCMAKE_BUILD_TYPE=${build_type}")
 
-echo "Testing AWS-LC in ${build_type} mode with address sanitizer."
-build_and_test -DASAN=1 -DUSE_CUSTOM_LIBCXX=1 "${cflags[@]}"
+if [ $(dpkg --print-architecture) == "arm64" ]; then
+  # BoringSSL provides two sets tests: the C/C++ tests and the blackbox tests.
+  # Details: https://github.com/google/boringssl/blob/master/BUILDING.md
+  # The blackbox tests (run `go test` under `ssl/test/runner`) takes 30 minutes to complete on ARM when ASAN clang flag enabled.
+  # This does not happen on X86 ASAN and ARM (when ASAN disabled).
+  # TODO: Add detailed reproduces steps and open a GitHub issue on https://github.com/google/sanitizers/issues
```

Comment:
May be you can add a sentence here and edit the TODO as follows:
Instead of `build_and_test`, only `all_tests.go` is run which runs the C/C++ tests and excludes the blackbox tests
TODO:
- reproduce the issue with the tests that take a long time to complete.
- open a GitHub issue https://github.com/google/sanitizers/issues specifically relating to BoringSSL.

Reply:
Updated. The issue draft is available in CryptoAlg-558.